cas-server-webapp-actions/src/main/java/org/jasig/cas/web/flow/AuthenticationViaFormAction.java

```diff
@@ -164,15 +160,14 @@ protected Event grantServiceTicket(final RequestContext context, final Credentia
             return newEvent(AbstractCasWebflowConfigurer.TRANSITION_ID_WARN);
 
         } catch (final AuthenticationException e) {
-            return newEvent(AUTHENTICATION_FAILURE, e);
-        } catch (final TicketCreationException e) {
-            logger.warn("Invalid attempt to access service using renew=true with different credential. Ending SSO session.");
-            this.centralAuthenticationService.destroyTicketGrantingTicket(ticketGrantingTicketId);
+            return newEvent(AbstractCasWebflowConfigurer.EVENT_AUTHENTICATION_FAILURE, e);
         } catch (final AbstractTicketException e) {
-            return newEvent(AbstractCasWebflowConfigurer.TRANSITION_ID_ERROR, e);
+            if (e instanceof TicketCreationException) {
+                logger.warn("Invalid attempt to access service using renew=true with different credential. Ending SSO session.");
```

Comment:
re-surfacing this comment since github consumed it with a nearby change.

This message already existed and was moved here from an earlier catch. The `TicketCreationException` itself is never even used within core CAS.